statsmodels/graphics/tsaplots.py

```diff
@@ -201,10 +202,10 @@ def seasonal_plot(grouped_x, xticklabels, ylabel=None, ax=None):
     ticks = []
     for season, df in grouped_x:
         df = df.copy() # or sort balks for series. may be better way
-        if pandas.__version__ < '0.17.0':
-            df.sort()
-        else:
+        if is_pandas_min_version(StrictVersion('0.17.0')):
```

Comment:
To be honest, I think the proper way to solve this would be to create a `pandas_sort` function that does the sorting in a uniform way, independent from the version of pandas or numpy.